src/widgets/study-with-us/ui/study-with-us.module.scss

```diff
@@ -0,0 +1,74 @@
+.study {
```

Comment:
Maybe we could improve the naming. At first glance it is difficult to understand what is study

Reply:
Done